route_optimizer/app/route_optimizer.py

```
@@ -0,0 +1,187 @@
+            distances = result["distances"]
+            
+            # numpy配列に変換
+            matrix = np.array(distances)
+            
+            # API制限回避の待機
+            time.sleep(API_WAIT_TIME)
+            
+            return matrix
+            
+        except Exception as e:
+            raise Exception(f"API呼び出しエラー: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)